Formula/counterfeiter.rb

```diff
@@ -16,12 +16,15 @@ class Counterfeiter < Formula
   depends_on "go"
 
   def install
-    system "go", "build", *std_go_args
+    system "go", "build", *std_go_args(ldflags: "-s -w")
   end
 
   test do
-    output = shell_output("#{bin}/counterfeiter -p os 2>&1", 1)
-    assert_match "Writing `Os` to `osshim`...", output
+    ENV["GOROOT"] = Formula["go"].opt_libexec
```

Comment:
If this needs to be set for `counterfeiter` to work we should set this in an env script. The env script should give users the option to override `GOROOT` if they want to set it to something else.

Reply:
Hmmm, maybe `counterfeiter` expects users to export the output of `go env` into their environment? In any case, I guess this should be fine.